### PR TITLE
Reduce db queries for faster page loading (#96) and change font size (#100)

### PR DESCRIPTION
**Pull request by nschneid**

### CODE REVIEW

File: wiki/plugins/metadata/static/wiki/css/metadata.css [hunk 1]
```diff
@@ -13,7 +13,7 @@ ul.inline,ul.inline > li { display: inline-block; }
 ul.inline { padding: 0; }
 ul.inline > li { margin: 0px 5px; }
 
-.wiki-article { font-family: 'Roboto', sans-serif; }
+.wiki-article { font-family: 'Roboto', sans-serif; font-size: 14pt; }
```
Comment (ablodge) [inline wiki/plugins/metadata/static/wiki/css/metadata.css]: Will this affect Construal, Usage, and Adposition articles?

Comment (nschneid) [reply wiki/plugins/metadata/static/wiki/css/metadata.css]: Yep, all articles. I figure it's easier to read the larger font.

File: wiki/plugins/metadata/tables.py [hunk 2]
```diff
@@ -101,7 +111,6 @@ def render_sentid(self, value):
 
     class Meta:
         model = PTokenAnnotation
-        #fields = ('id', 'adposition', 'construal', 'usage', 'sentence')
         fields = ('adp_pos', 'gov_head', 'gov_pos', 'gov_supersense', 'obj_head', 'obj_pos', 'obj_supersense', 'gov_obj_syntax', 'is_transitive')
```
Comment (ablodge) [inline wiki/plugins/metadata/tables.py]: What's this change for?

**Comment by ablodge** [comment]
Limits database queries by relying on `select_related`, change `get_absolute_url()` to property, and make articles large font.

Looks good to me!

File: wiki/plugins/metadata/tables.py (hunk 2 above)
Comment (nschneid) [reply wiki/plugins/metadata/tables.py]: Removing an old comment.

**Pull request merged**